src/restic/progress_unix.go

```diff
@@ -0,0 +1,28 @@
+// +build dragonfly linux netbsd openbsd freebsd solaris darwin
+
+package restic
+
+import (
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	"restic/debug"
+)
+
+// listen for incoming SIGUSR1 and print progress line
+var handleSIGUSR1 sync.Once
```

Comment:
Why is `sync.Once` used here? `init()` below will be run during initialisation exactly once...